input/pagecontent/use-cases.md

```diff
@@ -106,39 +98,34 @@ This page contains the use cases for AU eRequesting for the first release of the
-    <td>Requesting Practitioner (e.g. GP)</td>
-    <td>Requesting System<br>EHR</td>
-    <td>Provider Details<br>Token Identifier?<br>Patient Identifiers?</td>
+    <td><a href="ActorDefinition-au-erequesting-actor-placer.html">AU eRequesting Placer actor</a><br><a href="ActorDefinition-au-erequesting-actor-server.html">AU eRequesting Server actor</a></td>
+   <td>Service request</td>
   </tr>
   <tr>
     <td>G02</td>
     <td><strong>Cancelling an eRequest (requestor)</strong><br><br>An eRequest is revoked/cancelled by the requesting health care professional because the requested service is no longer required.</td>
-    <td>Requesting Practitioner (e.g. GP)</td>
-    <td>Requesting System<br>EHR</td>
-    <td>Provider Details<br>Token Identifier?<br>Patient Identifiers?</td>
+    <td><a href="ActorDefinition-au-erequesting-actor-placer.html">AU eRequesting Placer actor</a><br><a href="ActorDefinition-au-erequesting-actor-server.html">AU eRequesting Server actor</a></td>
+   <td>Service request</td>
   </tr>
   <tr>
     <td>G03</td>
     <td><strong>Cancelling an eRequest (provider)</strong><br><br>An eRequest is revoked/cancelled by the service provider because they are no longer able to provide the requested service for the patient. This cancellation has occurred before the provider has started the service (e.g. before the patient has attended the provider), hence it is not considered a non-fulfilment of the service.</td>
-    <td>Service Provider</td>
-    <td>EHR</td>
-    <td>Requestor Details<br>Token Identifier?<br>Patient Identifiers?</td>
+    <td><a href="ActorDefinition-au-erequesting-actor-filler.html">AU eRequesting Filler actor</a><br><a href="ActorDefinition-au-erequesting-actor-server.html">AU eRequesting Server actor</a></td>
+   <td>Service request</td>
   </tr>
   <tr>
     <td>G07</td>
     <td><strong>Specification of results to be copied to another clinician</strong><br><br>A hospital clinician requests diagnostic imaging for a patient after they have been discharged from the hospital, as a follow-up test. The results are requested to be sent to the hospital clinician and the patient's GP.</td>
-    <td>Hospital Clinician / Specialist<br>GP / Specialist<br>Service Provider</td>
-    <td>Requesting System<br>Service provider booking system</td>
-    <td>Patient Details<br>Requestor Details<br>Requested Service Details<br>Relevant Clinical Details<br>Billing Details<br>Results Copy To</td>
+    <td><a href="ActorDefinition-au-erequesting-actor-placer.html">AU eRequesting Placer actor</a><br><a href="ActorDefinition-au-erequesting-actor-server.html">AU eRequesting Server actor</a></td>
+   <td>Service request<br>Pathology test request<br>Implanted medical device summary<br>Adverse reaction risk summary<br>Problem/Diagnosis summary<br>Sex and gender summary</td>
   </tr>
 </tbody>
 </table>
```

Comment:
Add a link to the AUeReqDI mapping page above the table.

Reply:
Done